frontend/src/components/v-ramp.vue

```diff
@@ -2,20 +2,21 @@
 .ramp
   template(v-if="tframe === 'commit'")
     template(v-for="(slice, j) in user.commits")
-      a.ramp__slice(
-        draggable="false",
-        v-on:click="rampClick",
-        v-for="(commit, k) in slice.commitResults.filter(commitResult => commitResult.insertions > 0)",
-        v-bind:href="getLink(commit)", target="_blank",
-        v-bind:title="getContributionMessage(slice, commit)",
-        v-bind:class="'ramp__slice--color' + getSliceColor(slice.date)",
-        v-bind:style="{\
-          zIndex: user.commits.length - j,\
-          borderLeftWidth: getWidth(commit) + 'em',\
-          right: ((getSlicePos(slice.date)\
-            + (getCommitPos(k, slice.commitResults.length))) * 100) + '%'\
-          }"
-      )
+      template(v-for="(commit, k) in slice.commitResults.filter(commitResult => commitResult.insertions > 0)")
+        a.ramp__slice(
+          draggable="false",
+          v-on:click="rampClick",
+          v-bind:href="getLink(commit)", target="_blank",
+          v-bind:title="getLinkMessage(getLink(commit), getContributionMessage(slice, commit))",
```

Comment:
I think this one should still show the original contribution message. It is fine to show the commit title even when the ramp is not clickable.